packages/embark-coverage/src/lib/instrumenter.ts

```diff
@@ -28,10 +30,17 @@ export class Instrumenter {
       return;
     }
 
-    this.addInjectionPoints("contractDefinition", 1, node.loc);
+    this.addInjectionPoints("contractDefinition", 1, node);
   }
 
   public instrumentFunction(node: FunctionDefinition) {
+    // Remove stateMutability so that the function is not exported
+    // as "view" or "pure", seen as it will contain some emit
+    // statements and those require a transaction.
+    if (node.stateMutability === "view" || node.stateMutability === "pure") {
```

Comment:
No biggie, but would love to see `view` and `pure` in constants as well

Reply:
On it!